client/test/unit/specs/transactions.spec.js

```diff
@@ -47,7 +48,7 @@ describe('Testing Transactions.vue', () => {
       priorityCd: 'Y',
       request_action_cd: null,
       requestTypeCd: 'XUL',
-      stateCd: 'APPROVED',
+      stateCd: 'CONSUMED',
```

Comment:
Are there tests for EXPIRED and CONDITION? Do they still work OK? If not, please add some. Thx!

Reply:
This test is for consumed state. Test passes now with this change

Reply:
Please add a test for Expired (if there isn't one already). Thanks.

Reply:
Added